WcaOnRails/config/locales/en.yml

```diff
@@ -22,6 +22,27 @@
 en:
   #context: Special country name
   countries:
+    AG: "Antigua and Barbuda"
+    BA: "Bosnia and Herzegovina"
+    CD: "Democratic Republic of the Congo"
+    CG: "Congo"
+    CI: "Côte d'Ivoire"
+    CV: "Cabo Verde"
+    FM: "Federated States of Micronesia"
+    GW: "Guinea Bissau"
+    HK: "Hong Kong"
+    KN: "Saint Kitts and Nevis"
+    KP: "Democratic People’s Republic of Korea"
+    KR: "Republic of Korea"
+    LC: "Saint Lucia"
+    MM: "Myanmar"
+    MO: "Macau"
+    PS: "Palestine"
+    ST: "São Tomé and Príncipe"
+    TT: "Trinidad and Tobago"
+    VA: "Holy See"
+    VC: "Saint Vincent and the Grenadines"
+
```

Comment:
nit: any reason for the extra new line here?

Reply:
Ah, I added it to visually distinguish between real and "fictive" countries. Probably should have added a comment instead :p